src/App.js

```diff
@@ -2,6 +2,8 @@ import React, { useEffect } from 'react'
 import { Route, Switch } from 'wouter'
 import GlobalStyle from './theme/GlobalStyle'
 import ThemeProvider from './theme/ThemeProvider'
+import NavBar from './components/Navbar'
```

Comment:
Let's keep it consistent with the filename (And also change the rest of the `NavBar`s in this file to `Navbar`s
```suggestion
import Navbar from './components/Navbar'
```